### PR TITLE
Just domo

### PR DESCRIPTION
**Pull request by flamingo-peacock**
3rd time is a charm..... maybe?

### CODE REVIEW

File: apps/reducers/lib/services/domoget.ex [hunk 1]
```diff
@@ -53,6 +56,174 @@ defmodule Service.Domo do
...
+      _ ->
+        store_1_data = dataset[store_1]
+        store_2_data = dataset[store_2]
+
+        Map.drop(dataset, [store_1, store_2])
+        |> Map.put(store_1, store_1_data ++ store_2_data )
+    end
+  end
+
+  def merge_three_stores(dataset, _type, store_1, store_2, store_3) when is_list(dataset) do
```
Comment (randysecrist) [inline apps/reducers/lib/services/domoget.ex]: Could merge two & three stores be generically changed to just be "merge_stores"?

(When merging three stores (A,B, and C), could we make it simply two operations of merge stores?

e.g.
X = merge(A,B)
Y = merge(X, C)

File: apps/reducers/lib/services/stats.ex [hunk 2]
```diff
@@ -30,8 +30,9 @@ defmodule Service.Stats do
     data = event.meta["data"]
     store_num = event.meta["store"]
     type = String.to_existing_atom(event.type)
-    preped_data = data_prep(data, type)
-    apply(__MODULE__, type, [model, preped_data])
+    # preped_data = data_prep(data, type)
+    # apply(__MODULE__, type, [model, preped_data])
```
Comment (randysecrist) [inline apps/reducers/lib/services/stats.ex]: Comments still needed?

File: apps/reducers/lib/services/domoget.ex [hunk 3]
```diff
@@ -53,6 +56,174 @@ defmodule Service.Domo do
...
+  def get_timestamp({:pre_challenge, event})do
+    max = Enum.max_by(event.data["users"], fn(user) -> elem(user, 1)["start_time"] end)
+    elem(max, 1)["start_time"]
+  end
+  def get_timestamp({:refill_challenge, event})do
+    max = Enum.max_by(event.data["users"], fn(user) -> elem(user, 1)["start_time"] end)
+    elem(max, 1)["start_time"]
+  end
+
+  def date(datestring) when datestring == "", do: 1494287000000 #this date will be before season1 starts, so it won't get played
```
Comment (randysecrist) [inline apps/reducers/lib/services/domoget.ex]: May want to comment what the job of the date function is.  I can see the pattern match on a specific date which represents tournament start time.

But when looking at just this set of functions by itself, I can't really tell what should be happening.

Any idea how much test coverage (% lines covered) the domoget module has?

File: apps/api/mix.exs [hunk 4]
```diff
@@ -35,7 +35,7 @@ defmodule API.Mixfile do
       {:json, "~> 1.0"},
       {:exometer_core, github: "Feuerlabs/exometer_core"},
       {:setup, github: "uwiger/setup", manager: :rebar, override: true},
-      {:gen_stage, "~> 0.12.0"},
+      {:gen_stage, "~> 0.12.1"},
```
Comment (randysecrist) [inline apps/api/mix.exs:38]: Thank you for updating this.

File: apps/reducers/lib/mix/tasks/bracket.ex [hunk 5]
```diff
@@ -24,7 +24,7 @@ defmodule Mix.Tasks.Bracket do
     case response do
       {:ok, %HTTPoison.Response{status_code: 200, body: data}} ->
         {:ok, decoded_data} = Poison.decode(data)
-        {score, accuracy} = total_week(decoded_data["stats"][week])
+        {score, accuracy} = total_week(decoded_data[week])
```
Comment (randysecrist) [inline apps/reducers/lib/mix/tasks/bracket.ex:27]: Does the bracket task need to be run on next deploy?  Any others?

**Review by randysecrist — CHANGES_REQUESTED** [review]

File: apps/reducers/lib/mix/tasks/bracket.ex (hunk 5 above)
Comment (flamingo-peacock) [reply apps/reducers/lib/mix/tasks/bracket.ex:27]: no it doesn't, well it runs on a cron every sunday.

File: apps/api/mix.exs (hunk 4 above)
Comment (flamingo-peacock) [reply apps/api/mix.exs:38]: you're welcome

File: apps/reducers/lib/services/stats.ex (hunk 2 above)
Comment (flamingo-peacock) [reply apps/reducers/lib/services/stats.ex]: no, when moving things around I wanted to make sure i kept what was needed, obviously that isn't.

File: apps/reducers/lib/services/domoget.ex (hunk 3 above)
Comment (flamingo-peacock) [reply apps/reducers/lib/services/domoget.ex]: added some comments there!

**Review by randysecrist — APPROVED** [review]

**Pull request merged**